adagrams/game.py

```diff
@@ -1,11 +1,91 @@
+import random 
+
+LETTER_KEY = {
+    'A': 9,
+    'B': 2,
+    'C': 2,
+    'D': 4,
+    'E': 12,
+    'F': 2,
+    'G': 3,
+    'H': 2,
+    'I': 9,
+    'J': 1,
+    'K': 1,
+    'L': 4,
+    'M': 2,
+    'N': 6,
+    'O': 8,
+    'P': 2,
+    'Q': 1,
+    'R': 6,
+    'S': 4,
+    'T': 6,
+    'U': 4,
+    'V': 2,
+    'W': 2,
+    'X': 1,
+    'Y': 2,
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    letter_pool = []
+    for letter, frequency in LETTER_KEY.items():
+        for number in range(frequency):
+            letter_pool.append(letter)
+    
+    user_hand = []
+    while len(user_hand) < 10:
+        letter_to_add = random.choice(letter_pool)
+        if user_hand.count(letter_to_add) < LETTER_KEY[letter_to_add]:
+            user_hand.append(letter_to_add)
+    
+    return user_hand 
 
 def uses_available_letters(word, letter_bank):
-    pass
+    for letter in word: 
+        if word.count(letter) > letter_bank.count(letter):
+            return False 
+    return True
 
-def score_word(word):
-    pass
+def score_word(word): 
+    word = word.upper() 
+    score = 0
+    for letter in word:
+        if letter in ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T']:
+            score += 1 
+        elif letter in ['D', 'G']:
+            score += 2
+        elif letter in ['B', 'C', 'M', 'P']:
+            score += 3
+        elif letter in ['F', 'H', 'V', 'W', 'Y']:
+            score += 4
+        elif letter in ['K']:
+            score += 5
+        elif letter in ['J', 'X']:
+            score += 8
+        elif letter in ['Q', 'Z']:
+            score += 10 
```

Comment:
You could create a dictionary with these as key-value pairs. Or you could create a constant global value that holds this data and then do something like: 
```suggestion
    for letter in word:
      score += LETTER_VALUES[letter.upper()]
```